java/client/src/org/openqa/selenium/remote/CapabilityType.java

```diff
@@ -66,4 +68,13 @@
     String PROXY_PAC = "proxy_pac";
     String ENSURING_CLEAN_SESSION = "ensureCleanSession";
   }
+
+  /**
+   * AppDynamics extensions
+   *
+   * @author Renault John Lecoultre <renault.lecoultre@appdynamics.com>
+   * @since 7/1/15
+   * Copyright 2015 AppDynamics inc. All rights reserved
+   */
+  final String LOCK_STEP = "lockStep";
```

Comment:
LOCK_STEP seems too generic. How about LOCK_STEP_COMMAND_EXECUTION? Also, a brief overview of what it means would be helpful here. If not, at least a {@link} to WptHookAwareHandler would be good.